src/main/java/uk/gov/companieshouse/api/testdata/model/rest/CompanySpec.java

```diff
@@ -1,23 +1,31 @@
 package uk.gov.companieshouse.api.testdata.model.rest;
 
 import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
  * Requirements a new company must meet
- *
  */
 public class CompanySpec {
 
     @JsonProperty
-    @NotNull(message="invalid jurisdiction")
+    @NotNull(message = "invalid jurisdiction")
     private Jurisdiction jurisdiction;
 
     @JsonIgnore
     private String companyNumber;
 
+    @JsonProperty("company_status")
+    @Pattern(regexp = "active|inactive|closed|dissolved|administration|open|insolvency-proceedings|liquidation|converted-closed|receivership|registered|removed|voluntary-arrangement", message = "Invalid company status")
+    private String companyStatus;
+
+    @JsonProperty("type")
+    @Pattern(regexp = "assurance-company|charitable-incorporated-organisation|company_delta|converted-or-closed|eeig|eeig-establishment|european-public-limited-liability-company-se|fake-type|further-education-or-sixth-form-college-corporation|icvc-securities|icvc-umbrella|icvc-warrant|industrial-and-provident-society|invalid|investment-company-with-variable-capital|limited-partnership|llp|ltd|northern-ireland|northern-ireland-other|old-public-company|other|oversea-company|plc|private-limited-guarant-nsc|private-limited-guarant-nsc-limited-exemption|private-limited-shares-section-30-exemption|private-unlimited|private-unlimited-nsc|protected-cell-company|registered-overseas-entity|registered-society-non-jurisdictional|royal-charter|scottish-charitable-incorporated-organisation|scottish-partnership|uk-establishment|ukeig|united-kingdom-societas|unregistered-company", message = "Invalid company type")
```

Comment:
we could do with a unit test to test  "Invalid company type" and Invalid company status is thrown

Reply:
added methods for invalid company status and type